package.json

```diff
@@ -78,9 +78,9 @@
     "@babel/plugin-proposal-class-properties": "^7.1.0",
     "@emotion/core": "^10.0.22",
     "@emotion/styled": "^10.0.23",
-    "@hedviginsurance/brand": "^3.0.5",
+    "@hedviginsurance/brand": "3.0.5",
     "@hedviginsurance/cli": "^2.3.2",
-    "@hedviginsurance/embark": "^1.0.66",
+    "@hedviginsurance/embark": "^2.0.0-alpha.3",
```

Comment:
We have to get 1.0.67 into this version before going live with combo

Reply:
I merged that with `alpha.3` actually so i think it should work ™